File: fix-myaccount.py
f = r'c:\Users\acer\Desktop\Integracion\Intitech-Development-main\Intitech-Development-main\src\components\MyAccount\index.tsx'
with open(f, 'r', encoding='utf-8') as fp:
    content = fp.read()

old = "Dirección: {userProfile?.shippingAddress?.street ? `, ` : 'No guardada aún'}"
new = "Dirección: {userProfile?.shippingAddress?.street ? `${userProfile.shippingAddress.street}, ${userProfile.shippingAddress.city}` : 'No guardada aún'}"

count = content.count(old)
print('Matches:', count)
content = content.replace(old, new)
with open(f, 'w', encoding='utf-8') as fp:
    fp.write(content)
print('Done')

File: fix-myaccount.js
const fs = require('fs');
const f = 'c:/Users/acer/Desktop/Integracion/Intitech-Development-main/Intitech-Development-main/src/components/MyAccount/index.tsx';
let content = fs.readFileSync(f, 'utf8');

const streetCity = '`${userProfile.shippingAddress.street}, ${userProfile.shippingAddress.city}`';
const noGuardada = "'No guardada aún'";
const old = `Dirección: {userProfile?.shippingAddress?.street ? \`, \` : ${noGuardada}}`;
const rep = `Dirección: {userProfile?.shippingAddress?.street ? ${streetCity} : ${noGuardada}}`;

const count = content.split(old).length - 1;
console.log('Matches:', count);
content = content.split(old).join(rep);
fs.writeFileSync(f, content, 'utf8');
console.log('Done');
